pom.xml

```diff
@@ -67,6 +68,11 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
     <nexus.staging.maven.plugin.version>1.6.8</nexus.staging.maven.plugin.version>
     <lifecycle.mapping.version>1.0.0</lifecycle.mapping.version>
+
+    <!-- hcom specific plugin properties -->
+    <findbugs.config>file://${project.build.directory}//plugin-config/findbugs/hotels-findbugs-exclude.xml</findbugs.config>
```

Comment:
I think all these values should be relative to the `resources` folder in the `hotels-oss-plugin-config` parent as they will be unpacked that way (search for the `unpack` goal below). So this would become `findbugs/hotels-findbugs-exclude.xml`. Similar for the two below.

Reply:
I had a little test to make sure this time, and it seems like you have to provide the entire path from `${project.build.directory}` for spotbugs to be able to find the file, otherwise it throws this error: 
[ERROR] Could not find resource 'plugin-config/spotbugs/hotels-spotbugs-exclude.xml'.

Reply:
Yeah weird, I had a quick look now and it does what you say. I don't know how this works for our internal parent pom which seems to be doing some magic that I can't follow so you don't have to specify the `project.build.directory`. Oh well, if this works it's OK.